public/pages/workflows/workflows.tsx

```diff
@@ -277,7 +323,55 @@ export function Workflows(props: WorkflowsProps) {
             pageTitle={pageTitleAndDescription}
             bottomBorder={false}
           />
-          {dataSourceEnabled && dataSourceId === undefined ? (
+          {/**
+           * Local cluster issues: could be due to cluster being down, permissions issues,
+           * and/or missing plugins.
+           */}
+          {!dataSourceEnabled && connectionErrors ? (
+            <EuiPageContent grow={true}>
+              <EuiEmptyPrompt
+                title={<h2>Error accessing cluster</h2>}
+                body={
+                  <p>
+                    Ensure your OpenSearch cluster is available and has the Flow
+                    Framework and ML Commons plugins installed.
+                  </p>
+                }
+                actions={
+                  <EuiButton color="primary" fill={false}>
+                    <EuiLink target="_blank" href={MAIN_PLUGIN_DOC_LINK}>
```

Comment:
EuiButton already supports `href` directly, no need to nest it with EuiLink? Line 364 below has the correct usage

Reply:
Originally changed since I needed the popout icon since I've set `target=_blank`. Utilized icon params exposed from `EuiButton` instead and removed `EuiLink`, thanks for the suggestion!